html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="https://example.com/register" method="post">
+      <label for="email">Email</label>
+      <br />
+      <input type="email" id="email" name="email" required />
+      <br /><br />
+      <label for="coding">Why is coding cool</label>
+      <br />
+      <select id="coding" name="coding">
+        <option value="">Please Select</option>
+        <option value="fun">Because coding is fun</option>
+        <option value="interesting">Because coding is interesting</option>
+        <option value="powerful">Because coding is powerful</option>
+      </select>
+      <br /><br />
+      <label>Favorite Language (Choose One)</label>
+      <br /><br />
+      <label><input type="radio" name="favorite" value="html" />HTML</label>
+      <label><input type="radio" name="favorite" value="css" />CSS</label>
+      <label
+        ><input
+          type="radio"
+          name="favorite"
+          value="javascript" />JavaScript</label
+      >
+      <br /><br />
+      <label>Would you like to sign up for more information</label>
+      <br /><br />
```

Comment:
If this does not connect to anywhere specific, this would just be text content and should be a <p> element instead of a <label> element.